src/ui-lib/sass/05-components/03-organisms/_hero.scss

```diff
@@ -0,0 +1,21 @@
+.grav-c-hero {
+  background-blend-mode: hard-light;
+  background-color: $grav-co-accent-secondary-dark;
+  background-size: cover;
+  background-position-x: center;
+  color: $grav-co-bg;
+  text-align: center;
+  padding: $grav-sp-xl 0;
+
+  @media all and (max-width: gravy-breakpoint(regular)) {
+    background-image: none !important;
+  }
+
```

Comment:
I think we should see if we can achieve this with an inline image. I really like the blend mode effect and think I should try it on the locations page. I could look into refactoring it?

Reply:
the inline image is going to have even more problems that we currently have. as I told you via voice, the use of the image will only complicate things,I think it's OK for now, I don't really want to meddle too much around it.

Reply:
Yeah for now it's fine, it shares some similarities with the tiles on the locations page so that's why I was wondering if we can align the two.